tasks/columns.py

```diff
@@ -0,0 +1,60 @@
+import json
+from collections import OrderedDict
+
+
+CONDITIONS = "conditions"
+EXCEPTIONS = "exceptions"
+
+
+class ColumnsDeclarations:
+    def __init__(self, JSONFile):
+        with open(JSONFile) as file:
+            self._columns = json.load(file)
+
+    def _find_column(self, colid):
+        return self._columns[colid]
+
+    def _get_requirements(self, colid, requirement):
+        try:
+            return self._find_column(colid)[requirement]
+        except:
+            return None
```

Comment:
Just do `return self._find_column(colid).get(requirement)` which is the same without the `try` block.